packages/vite/src/node/cli.ts

```diff
@@ -9,7 +9,6 @@ import type { ServerOptions } from './server'
 import type { CLIShortcut } from './shortcuts'
 import type { LogLevel } from './logger'
 import { createLogger } from './logger'
```

Comment:
I didn't do the same for `createLogger` as the file itself is small and I didn't notice any perf improvement if refactored as a dynamic import.